l3kernel/l3seq.dtx

```diff
@@ -189,6 +189,55 @@
 %   \end{texnote}
 % \end{function}
 %
+% \begin{function}[added = 2024-11-26]
+%   {
+%     \seq_set_regex_extract:Nnn, \seq_set_regex_extract:cnn,
+%     \seq_set_regex_extract:NNn, \seq_set_regex_extract:cNn,
+%     \seq_gset_regex_extract:Nnn, \seq_gset_regex_extract:cnn,
+%     \seq_gset_regex_extract:NNn, \seq_gset_regex_extract:cNn,
+%   }
+%   \begin{syntax}
+%     \cs{seq_set_regex_extract:Nnn} \meta{seq~var} \Arg{regex} \Arg{token list}
+%     \cs{seq_set_regex_extract:NNn} \meta{seq~var} \meta{regex~var} \Arg{token list}
+%   \end{syntax}
+%   Finds the first match of the \meta{regular expression} in the
+%   \meta{token list}. If it exists, the match is stored as the first
+%   item of the \meta{seq~var}, and further items are the contents of
+%   capturing groups, in the order of their opening parenthesis. If
+%   there is no match, the \meta{seq~var} is cleared.
```

Comment:
Replaced with the doc for `\regex_extract_all:...`, since currently `\seq_set_regex_extract:...` wraps `all` versions.

```suggestion
%   Finds all matches of the \meta{regular expression} in the
%   \meta{token list}, and stores all the submatch information in a
%   single sequence (concatenating the results of multiple
%   \cs{regex_extract_once:nnN} calls).  If there is no match, the
%   \meta{seq~var} is cleared.
```